library/packages/src/modules/PackageSystem.rb

```diff
@@ -107,11 +105,8 @@ def EnsureSourceInit
         EnsureTargetInit()
       end
 
-      sources = Pkg.SourceStartCache(true)
```

Comment:
so if i got it correctly this is the fix. Doesn't just deleting it influence running system use case?

Reply:
No, the running system is handled in the [code above](https://github.com/yast/yast-yast2/pull/1198/files#diff-29b7e878d71aa5590e4369adc14dfb84d31a05246bc2496734a6c5d84559c064R95-R99), there is another `Pkg.SourceStartCache` call.

Reply:
And I have tested that in an installed system to verify that it works fine.